pkg/oc/cli/newapp/newapp.go

```diff
@@ -344,7 +344,18 @@ func (o *AppOptions) RunNewApp() error {
 		}
 
 		if o.Action.ShouldPrint() {
-			return o.Printer.PrintObj(result.List, o.Out)
+			// TODO(juanvallejo): this needs to be fixed by updating QueryResult.List to be of type corev1.List
```

Comment:
Discussed with @juanvallejo, this will be fixed in a followup.

Reply:
Tracked in https://github.com/openshift/origin/issues/20919